pcsx2/gui/FrameForGS.cpp

```diff
@@ -167,19 +167,21 @@ void GSPanel::DoResize()
 		switchAR = false;
 	}
 
-	if (!switchAR) {
-		if (g_Conf->GSWindow.AspectRatio == AspectRatio_4_3)
+	if (switchAR) {
+		if (g_Conf->GSWindow.FMVAspectRatioSwitch == FMV_AspectRatio_Switch_4_3) {
 			targetAr = 4.0 / 3.0;
-		else if (g_Conf->GSWindow.AspectRatio == AspectRatio_16_9)
+		} else if (g_Conf->GSWindow.FMVAspectRatioSwitch == FMV_AspectRatio_Switch_16_9) {
 			targetAr = 16.0 / 9.0;
-	} else if (switchAR) {
-		if (g_Conf->GSWindow.FMVAspectRatioSwitch == FMV_AspectRatio_Switch_4_3 && g_Conf->GSWindow.AspectRatio != AspectRatio_4_3) {
+		} else {
+			// Allows for better real time toggling, returns to the non fmv override aspect ratio.
+			switchAR = false;
+		}
+	} else if (!switchAR) {
```

Comment:
@lightningterror Do you notice what is wrong with this else if condition?

Reply:
switchAR is disabled and could just be changed to a normal else since if (switchAR) checks if it's enabled ?

Reply:
Yes.

I kinda prefer pointing things out before they're merged, not afterwards. It'd be nice if you could actually leave PRs open for longer to give people time to actually review things.

Reply:
I admit I've been kinda rushing to get things merged faster, just so we could focus on other work without maintaining multiple prs conflict free.
I'll keep them open a bit longer from now on.

I'll update that check on the next wx pr I open.